lms/envs/test.py

```diff
@@ -450,7 +450,7 @@
 FEATURES['ENABLE_EDXNOTES'] = True
 
 # Add milestones to Installed apps for testing
-INSTALLED_APPS += ('milestones', )
+INSTALLED_APPS += ('milestones', 'openedx.core.djangoapps.call_stack_manager.tests')
```

Comment:
This should just point to `openedx.core.djangoapps.call_stack_manager`, since that's the app.